<launch>

	<node pkg="socketcan_bridge" type="socketcan_bridge_node" name="socketcan_bridge_node" />

	<node pkg="gazebo_plugin" type="joint_trajectory_client" name="joint_trajectory_client" />
	
	<node pkg="can_comm" type="can_master_node" name="can_master_node" />

</launch> 
